v3/lints/cabf_ev/lint_ev_invalid_business_category.go

```diff
@@ -0,0 +1,69 @@
+/*
+ * ZLint Copyright 2024 Regents of the University of Michigan
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+ * implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+/*
+ * Contributed by Adriano Santoni <adriano.santoni@staff.aruba.it>
+ * of ACTALIS S.p.A. (www.actalis.com).
+ */
+
+package cabf_ev
+
+import (
+	"github.com/zmap/zcrypto/x509"
+	"github.com/zmap/zlint/v3/lint"
+	"github.com/zmap/zlint/v3/util"
+)
+
+func init() {
+	lint.RegisterCertificateLint(&lint.CertificateLint{
+		LintMetadata: lint.LintMetadata{
+			Name:          "e_ev_invalid_business_category",
+			Description:   "Checks that businessCategory contains a valid value as per EV Guidelines 9.2.3",
+			Citation:      "EVGs 9.2.3",
```

Comment:
I think this is supposed to be 7.1.4.2.3? https://github.com/cabforum/servercert/blob/main/docs/EVG.md#71423-subject-business-category-field

Reply:
I see that this JUST changed though https://github.com/cabforum/servercert/commit/c4a34fe2292022e0a04ba66b5a85df75907ac2a2

Reply:
@aaomidi , the requirement is still there, identical, although its location within the EVG has just changed as a consequence of reformatting the EVG according to RFC3647 (the goal of ballot SC65). So the relevant section changed from 9.2.3 to 7.1.4.2.3. if you prefer, I could refer to this latter in the `Citation` field, but the requirements has always existed.... so I am unsure what's best.

Reply:
Oh yeah I agree, I do think though that since the citation has changed, and this PR hasn't merged yet, it would make sense to cite the current location in the document?

Reply:
I changed the reference to 7.1.4.2.3